editor/src/components/context-menu-wrapper.tsx

```diff
@@ -78,17 +88,30 @@ export class MomentumContextMenu<T> extends ReactComponent<ContextMenuProps<T>>
     return splitItems
   }
 
+  isHidden = (
```

Comment:
this feels to speceific to live in the generic MomentumContextMenu – isn't there a more specific place to put it for the canvas context menu?

Reply:
this could be similar to how action callbacks are done for context menu, I'm not sure if that's better?

Reply:
I think that sounds better – something like an optional `isHidden?: (props: ContextMenuInnerProps) => void` property on the `ContextMenuItem` would mean we only need to provide it for the Canvas context menu, so we could remove the specific hiding code from here